noobaa/noobaa-core
# Revert "Bumping isa-l to v2.31.0"

### Explain the changes

This reverts commit 8a472c69b7683bacaa455fbac2467ab7639d4338.
We are facing an issue with Linux/arm64. it needs closer attention. 

### Testing Instructions:
1. run the build on Linux arm64 and see it builds:
`CONTAINER_PLATFORM=linux/arm64 make noobaa`
